apps/site/src/app/schedule/sections/SchedulePage.tsx

```diff
@@ -0,0 +1,140 @@
+"use client";
+
+import { useState, useEffect, useRef } from "react";
+import * as Tabs from "@radix-ui/react-tabs";
+
+import EventCard from "../components/EventCard";
+import convertToPST from "@/lib/utils/convertToPST";
+
+import "@radix-ui/themes/styles.css";
+import "./SchedulePage.scss";
+
+const T_REFRESH = 15000;
+
+const weekdayFormat = new Intl.DateTimeFormat("en", {
+	weekday: "long",
+});
+
+interface ScheduleProps {
+	schedule: {
+		title: string;
+		eventType: string;
+		location?: string | undefined;
+		virtual?: string | undefined;
+		startTime: Date;
+		endTime: Date;
+		organization?: string | undefined;
+		hosts?: string[] | undefined;
+		description: JSX.Element;
+	}[][];
+}
+
+export default function SchedulePage({ schedule }: ScheduleProps) {
+	const tabsRef = useRef<HTMLDivElement>(null);
+
+	const tabsOffsetTopRef = useRef<number | undefined | null>(null);
+
+	const [day, setDay] = useState("Friday");
+
+	const [now, setNow] = useState<Date>(convertToPST(new Date()));
+
+	const [haveTabsBeenScrolledToTop, setHaveTabsBeenScrolledToTop] =
+		useState(false);
+
+	useEffect(() => {
+		const refreshNow = setInterval(() => {
+			setNow(convertToPST(new Date()));
+		}, T_REFRESH);
+
+		return () => {
+			clearInterval(refreshNow);
+		};
+	}, []);
+
+	useEffect(() => {
+		tabsOffsetTopRef.current = tabsRef.current?.offsetTop;
+
+		const detectScroll = () => {
+			if (
+				tabsOffsetTopRef.current !== undefined &&
+				tabsOffsetTopRef.current !== null &&
+				window.scrollY > tabsOffsetTopRef.current
+			) {
+				setHaveTabsBeenScrolledToTop(true);
+			} else {
+				setHaveTabsBeenScrolledToTop(false);
+			}
+		};
+
+		window.addEventListener("scroll", detectScroll);
+
+		return () => {
+			window.removeEventListener("scroll", detectScroll);
+		};
+	}, []);
+
+	return (
+		<Tabs.Root
+			value={day}
+			defaultValue="Friday"
+			onValueChange={setDay}
+			className="w-11/12 sm:w-4/5"
+		>
```

Comment:
Thought/question: Did we confirm with graphics that we're doing this new tabbed approach? For reference, a few years back we did "anchor navigation" with "scroll spy", and last year we just had a long list with no separations.

Reply:
This is what was on the Figma, so that was what Albert proceeded with. I can double check with graphics.

Reply:
Graphics just said yes